### PR TITLE
Run scan logic if github secret and lpvs secret are the same

### PR DESCRIPTION
**Pull request by t25kim**
# Description

Scanning will be carried out when github secret and lpvs secret are the same.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?
1. Add `github.secret` to the `application.properties` file like below.
```
github.secret=lpvs
```
2. Run LPVS
3. Check the LPVS log like below.
```
2022-08-31 16:48:31.188  INFO 71382 --- [nio-7896-exec-1] c.l.controller.GitHubWebhooksController  : lpvs   signature: 42c493a2e4062a2fb506083432e1e9208e44000acfd04971097ac62a1779a0d5
2022-08-31 16:48:31.188  INFO 71382 --- [nio-7896-exec-1] c.l.controller.GitHubWebhooksController  : github signature: 42c493a2e4062a2fb506083432e1e9208e44000acfd04971097ac62a1779a0d5
```

**Test Configuration**:
* Java: v11
* LPVS Release: v1.0.0

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by t25kim** [comment]
@o-kopysov LGTM does not seem to support OpenJDK 11. Is there a minimum version of jdk that needs to be supported?

**Comment by o-kopysov** [comment]
> > I think that corresponding changes about `github.secret` have to be added to README and `application.properties` files.
> 
> @o-kopysov Please review my update.

Looks good to me. But let's wait for the feedback from @tdrozdovsky  about LGTM analysis failure.

**Comment by MoonkiHong** [comment]
https://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

File: src/main/java/com/lpvs/controller/GitHubWebhooksController.java [hunk 1]
```diff
@@ -37,17 +44,20 @@ public class GitHubWebhooksController {
...
 
     @RequestMapping(value = "/webhooks", method = RequestMethod.POST)
-    public ResponseEntity<ResponseWrapper> gitHubWebhooks(@RequestHeader(SIGNATURE) String signature, @RequestBody String payload) throws InterruptedException {
+    public ResponseEntity<ResponseWrapper> gitHubWebhooks(@RequestHeader(SIGNATURE) String signature, @RequestBody String payload) throws Exception {
         LOG.info("New webhook request received");
 
         // if signature is empty return 401
         if (!StringUtils.hasText(signature)) {
             return new ResponseEntity<>(new ResponseWrapper(ERROR), HttpStatus.FORBIDDEN);
+        } else if (!GITHUB_SECRET.isBlank() && wrongSecret(signature, payload)) {
```
Comment (o-kopysov) [inline src/main/java/com/lpvs/controller/GitHubWebhooksController.java:59]: May we try to change this line as follows?

```suggestion
        } else if (!GITHUB_SECRET.trim().isEmpty() && wrongSecret(signature, payload)) {
```

**Pull request merged**